Test/AbstractBlockServiceTestCase.php

```diff
@@ -55,6 +55,13 @@ protected function setUp()
         $this->blockContextManager = new BlockContextManager($blockLoader, $this->blockServiceManager);
     }
 
+    /**
+     * Create a mocked block service.
+     *
+     * @param BlockServiceInterface $blockService A block service
+     *
+     * @return BlockContextInterface
+     */
```

Comment:
canyou please do the phpdoc change at least in a seperate commit? thank you @core23 👍